src/main/java/net/sf/jabref/util/Util.java

```diff
@@ -665,6 +664,66 @@ public static String getResultsWithEncoding(URLConnection source, Charset encodi
     }
 
     /**
+     * Get the results of HTTP post on a URL and return contents as a String.
+     *
+     * @param source postData encoding
+     * @return
+     * @throws IOException
+     */
+    public static String getPostResultsWithEncoding(URL source, String postData, Charset encoding) throws IOException {
+        HttpURLConnection con = (HttpURLConnection) source.openConnection();
+
+        //add a default request header
+        con.setRequestMethod("POST");
+        con.setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 5.1; rv:31.0) Gecko/20100101 Firefox/31.0");
+        con.setRequestProperty("Accept-Language", "en-US,en;q=0.5");
+
+        return net.sf.jabref.util.Util.getPostResultsWithEncoding(con, postData, encoding);
```

Comment:
You can just call return getPostResult... as you call a method within this class.